### PR TITLE
Updating the SDK to the latest.

### PR DESCRIPTION
**Pull request by livarcocc**
@eerhardt @piotrpMSFT @jgoshi @krwq

### CODE REVIEW

**Comment by eerhardt** [comment]
One of the failing tests says:

```
Done Building Project \"D:\\j\\workspace\\release_windo---4c3b899b\\test\\Microsoft.DotNet.Cli.Utils.Tests\\bin\\Release\\netcoreapp1.0\\It_resolves_desktop_apps_when_configuration_is_Release\\AppWithProjTool2Fx\\App.csproj\" (Build target(s)) -- FAILED.\r\nBuild FAILED.\r\n\"D:\\j\\workspace\\release_windo---4c3b899b\\test\\Microsoft.DotNet.Cli.Utils.Tests\\bin\\Release\\netcoreapp1.0\\It_resolves_desktop_apps_when_configuration_is_Release\\AppWithProjTool2Fx\\App.csproj\" (Build target) (1) ->\r\n\"D:\\j\\workspace\\release_windo---4c3b899b\\test\\Microsoft.DotNet.Cli.Utils.Tests\\bin\\Release\\netcoreapp1.0\\It_resolves_desktop_apps_when_configuration_is_Release\\AppWithProjTool2Fx\\App.csproj\" (Build target) (1:3) ->\r\n(CheckRuntimeIdentifier target) -> \r\n D:\\j\\workspace\\release_windo---4c3b899b\\.nuget\\packages\\microsoft.net.sdk\\1.0.0-alpha-20161029-1\\build\\Microsoft.NET.RuntimeIdentifierInference.targets(45,5): error : RuntimeIdentifier must be set for .NETFramework executables. Consider RuntimeIdentifier=win7-x86 or RuntimeIdentifier=win7-x64. [D:\\j\\workspace\\release_windo---4c3b899b\\test\\Microsoft.DotNet.Cli.Utils.Tests\\bin\\Release\\netcoreapp1.0\\It_resolves_desktop_apps_when_configuration_is_Release\\AppWithProjTool2Fx\\App.csproj]\r\n 0 Warning(s)\r\n 1 Error(s)\r\nTime Elapsed 00:00:03.66\r\n\r\nStdErr:\r\n\r\n
```

`RuntimeIdentifier must be set for .NETFramework executables. Consider RuntimeIdentifier=win7-x86 or RuntimeIdentifier=win7-x64.`

Which is a new check that @nguerrera added last week.

**Comment by nguerrera** [comment]
Yes, mentioned this to @livarcocc offline

File: src/Microsoft.DotNet.TestFramework/TestAssetInstance.cs [hunk 1]
```diff
@@ -67,6 +69,27 @@ public TestAssetInstance WithBuildFiles()
             return this;
         }
 
+        public TestAssetInstance WithNuGetConfig(string nugetCache)
+        {
+            var thisAssembly = typeof(TestAssetInstance).GetTypeInfo().Assembly;
+            var newNuGetConfigPath = Path.Combine(Root.FullName, "NuGet.config");
```
Comment (piotrpMSFT) [inline src/Microsoft.DotNet.TestFramework/TestAssetInstance.cs]: Root.GetFile("Nuget.config");

**Comment by livarcocc** [comment]
@dotnet-bot Test Windows_NT x64 Release Build please

**Pull request merged**